pegasus/sites.v3/code.org/public/administrators.haml

```diff
@@ -20,12 +19,13 @@ theme: responsive_full_width
       %img.rounded-corners.box-shadow-primary{src: "/images/admins-page-top.png", alt: "", style: "position: relative; width: 100%;"}
     .clear
 
-- if !!DCDO.get('curriculum-launch-2024', false)
-  = view :curriculum_launch_skinny_banner
-- else
-  = view :section_divider_line
+-# Show Curriculum Launch banner if dcdo is true
```

Comment:
I'm going to update the dcdo flag used here to  be more evergreen so will update this comment to include the dcdo flag when I do.

Reply:
Sounds good!